src/webapi/rendering_context.rs

```diff
@@ -18,6 +20,160 @@ pub trait RenderingContext {
 #[reference(instance_of = "CanvasRenderingContext2D")]
 pub struct CanvasRenderingContext2d(Reference);
 
+/// The CanvasGradient interface represents an opaque object describing a gradient. 
+/// It is returned by the methods CanvasRenderingContext2D.createLinearGradient() or 
+/// CanvasRenderingContext2D.createRadialGradient().
```

Comment:
Very minor nitpick - these docs could use some light editing. (e.g. the MDN docs do say "interface", but in Rust this is somewhat confusing where by "interface" you'd usually think of traits, and these are concrete structures.)